app/src/main/java/com/hedvig/app/feature/birthday/viewmodel/BirthdayViewModel.kt

```diff
@@ -0,0 +1,59 @@
+package com.hedvig.app.feature.birthday.viewmodel
+
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.hedvig.app.feature.birthday.data.BirthdayRepository
+import io.reactivex.disposables.CompositeDisposable
+import io.reactivex.rxkotlin.plusAssign
+import timber.log.Timber
+import java.util.*
+
+
+class BirthdayViewModel(
+    private val birthdayRepository: BirthdayRepository
+) : ViewModel() {
+
+    private lateinit var birthDate: String
+
+    val isBirthdayData: MutableLiveData<Boolean> = MutableLiveData()
+    private val disposables = CompositeDisposable()
+
+
+
+    private fun isBirthday(): Boolean {
+
+        val birthMonth = birthDate.substring(4, birthDate.length - 6).toInt()
+        val birthDay = birthDate.substring(6, birthDate.length - 4).toInt()
+
+
+        val currentMonth = Calendar.getInstance().get(Calendar.MONTH) + 1
```

Comment:
you should be able to create one Calendar for this. Try to be restrictive with creating resources. Phones are way better now then apilevel 1 but we should still try to do as lean app as possible